Cookbook-ios/googlePlaces/Full Screen default/PlacesViewController.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  PlacesViewController.swift
+//  Cookbook-ios
+//
+//  Created by Diego Quiros on 06/04/2020.
+//  Copyright © 2020 Wolox. All rights reserved.
+//
+
+import UIKit
+import GooglePlaces
+
+// For more information visit the places developers documentation at
+//https://developers.google.com/places/ios-sdk/autocomplete#add_an_autocomplete_ui_control
+
+class PlacesViewController: GeneralViewController {
+    @IBOutlet weak var placeLabel: UILabel!
+    
+    @IBAction func onSelectPlacePressed(_ sender: Any) {
+        let autocompleteController = GMSAutocompleteViewController()
```

Comment:
Do you need to create a new instance each time the button is pressed?  (or is this a singleton?)

Reply:
I could create a lazy var with the controller to create it only once